packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -7,7 +7,14 @@
 
 import { AuthInfo, Connection } from '@salesforce/core';
 import * as vscode from 'vscode';
-import {  WorkspaceContextUtil } from '../../../src';
+import { ConfigAggregatorProvider, WorkspaceContextUtil } from '../../../src';
+import { AuthUtil } from '../../../src/auth/authUtil';
+jest.mock('@salesforce/core');
+jest.mock('../../../src/auth/authUtil');
+
+const authInfoMock = jest.mocked(AuthInfo);
+const connectionMock = jest.mocked(Connection);
+const authUtilMock = jest.mocked(AuthUtil);
```

Comment:
create typed mocks of the things we care about to interact with the mocked modules in tests.